automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -381,6 +381,64 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
     testResult.get
   }
 
+  def withLocalizeDelocalizeFiles[T](cluster: Cluster, localizeFileName: String, localizeFileContents: String, delocalizeFileName: String, delocalizeFileContents: String)(testCode: (Map[String, String], GcsBucketName) => T)(implicit webDriver: WebDriver, token: AuthToken): T = {
```

Comment:
I wonder if it may be easier to make sense of (especially for a newbie) if the variables are names as opposed to verbs (e.g. `localizedFileName` or `fileToLocalize` instead of `localizeFileName`, etc.).